CONTRIBUTING.md

```diff
@@ -9,46 +9,94 @@ To propose improvements, feel free to submit a PR or open an Issue.
 
 Many great ideas for new features come from the community, and we'd be happy to consider yours 👍.
 
-To share your idea or request, [open a GitHub Issue](https://github.com/DataDog/dd-sdk-ios/issues/new)  using dedicated issue template.
+To share your idea or request, [open a GitHub Issue](https://github.com/DataDog/dd-sdk-ios/issues/new) using dedicated issue template.
 
 ## Found a bug?
 
 For any urgent matters (such as outages) or issues concerning the Datadog service or UI, contact our support team via https://docs.datadoghq.com/help/ for direct, faster assistance.
 
-You may submit a bug report concerning the Datadog SDK for iOS by [opening a GitHub Issue](https://github.com/DataDog/dd-sdk-android/issues/new). Use dedicated bug-issue template and provide all listed details to let us solve it better. 
-
+You may submit a bug report concerning the Datadog SDK for iOS by [opening a GitHub Issue](https://github.com/DataDog/dd-sdk-android/issues/new). Use dedicated bug-issue template and provide all listed details to let us solve it better.
 
 ## Have a patch?
 
 We welcome all code contributions to the library. If you have a patch adding value to the SDK, let us know 💪! Before you [submit a Pull Request](https://github.com/DataDog/dd-sdk-ios/pull/new/master), make sure that you first create an Issue to explain the bug or the feature your patch covers, then make sure similar Issue or PR doesn't already exist.
 
 Your Pull Request will be run through our CI pipeline, and a project member will review the changes with you. At a minimum, to be accepted and merged, Pull Requests must:
- - have a stated goal and detailed description of the changes made;
- - include thorough test coverage and documentation, where applicable;
- - pass all tests and code quality checks on CI;
- - receive at least one approval from a project member with push permissions.
+
+- have a stated goal and detailed description of the changes made;
+- include thorough test coverage and documentation, where applicable;
+- pass all tests and code quality checks on CI;
+- receive at least one approval from a project member with push permissions.
 
 Make sure that your code is clean and readable, that your commits are small and atomic, with a proper commit message.
 
-### Getting started
+## Shall we start?
+
+### _NOTE:_ The easiest way to start is to run `make` command
+
+<details>
+  <summary>Makefile</summary>
+
+- the most common build operations defined in this file
+- CI always use `make` commands
+  </details>
+
+<details>
+<summary>Datadog/</summary>
+
+- Datadog.xcodeproj
+  - Xcode project containing Datadog SDK
+- Datadog
+  - Source files of SDK
+- DatadogTests
+  - Unit tests of SDK
+- DatadogObjC
+  - ObjC wrapper of SDK
```

Comment:
basically, if we have 3 targets (Datadog, DatadogObjC, DatadogTests) it makes sense to me to group them in the same way. should we do it differently?

sure, no need to explain 👍 
actually i left that boilerplate in order to discuss over the file; but that was a bad idea as we could discuss with PR comments.

Reply:
I mean, it's OK for me to structure `Datadog/*` folder as we propose here 🙂. I only mean, to not explain it further inside `CONTRIBUTING.md`, as this will be another effort to maintain if thing do change.